recipe/meta.yaml

```
@@ -10,6 +10,7 @@ source:
   sha256: b4c825c790f28c619c176044c9e8836b7ca4a5125b93b2b9f8ee9039f16b21c3
 
 build:
+  noarch: python
   number: 0
```

Bump the build number?
```suggestion
  number: 1
```

---

Do we need to? Since the dependencies are the same as #11 right?

---

https://conda-forge.org/docs/maintainer/updating_pkgs.html#bumping-the-build-number

> The build number is increased when the source code for the package has not changed but you need to make a new build. As a rule of thumb, the build number is increased whenever a new package with the same version needs to be uploaded to the conda-forge channel.

I think we have to. Otherwise there may be some conflicts, as pygmt v0.2.1 build 0 will have both OS-specific packages and OS-independent packages.

---

Ok, best be safe then.